README.md

```diff
@@ -212,14 +213,34 @@ Each time you change the Dockerfile or dependencies it mounts (e.g. the Gemfile)
 
 ### Customizing Git integration
 
-The development environment uses the [v3 repository](https://github.com/exercism/v3) as its source for track contents, such as its exercises, concepts and documentation.
+The development environment uses the [v3 repository](https://github.com/exercism/v3) as its source for track contents, such as its exercises, concepts and documentation. 
 
-There are three environment variables you can use to customize the Git integration:
+If you are using the development environment to work on a specific track, replace the v3 repository with the track you are working on. 
```

Comment:
```suggestion
If you are using the development environment to work on a specific track, you can instruct the website to use a specific repository (e.g. a local fork) rather than downloading the default repositories from GitHub. 
```